README.md

```diff
@@ -277,7 +321,7 @@ What follows is a deeper dive into both of these content types.
 
 For any kind of repeating content, this is the basic process:
 
-1. In the vars file for the section you're in (e.g., `blog/_vars-blog.html`),
+1. In the vars file for the section you're in (e.g., `newsroom/_vars-newsroom.html`),
```

Comment:
Events are wagtail now as well. From what I understand, the only bit that's not in WP at this time is the Press Resources page, so I guess the newsroom vars are still used there, though we might as well move them to that template. I've opened an issue in the Jira backlog to clean out the old WP driven templates.